### DISCUSSION TITLE
Copy Names to Label Columns in Saved Searches

### DISCUSSION THREAD

**Discussion by WildWombat2000** [burkybang/SuiteAdvanced-Feedback#168] [Ideas]
Just an idea - when creating saved searches, it would be very useful to have some sort of button to copy the Field names (first column) to the Summary Label column. Time consuming in certain summary searches where you are mostly using Max/Min as summary types.
![Screenshot 2024-08-15 085649](https://github.com/user-attachments/assets/bc8f15fd-e729-40e5-9278-2a0b16241a7f)

**Comment by burkybang** [comment]
That's a good idea!

**Reply by WildWombat2000** [reply]
Hoping this is still on your hitlist somewhere? Would be great to have a button 'Default Summary Labels'. May be too niche - LMK.

**Reply by burkybang** [reply]
Yes! Will take a look soon
